### PR TITLE
test(contact): add a test that sends a contact request  from the chat

### LINKED ISSUE #6883: [Automation] Can create a contact request

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #6883

### CODE REVIEW

File: test/ui-test/testSuites/suite_onboarding/shared/steps/signUpSteps.py [hunk 1]
```diff
@@ -11,7 +11,11 @@
 
 @Given("A first time user lands on the status desktop and generates new key")
 def step(context):
-    init_steps.a_first_time_user_lands_on_and_generates_new_key(context)    
+    init_steps.a_first_time_user_lands_on_and_generates_new_key(context)
+
+@Given("A user lands on the status desktop and generates new key")
```
Comment (endulab) [inline test/ui-test/testSuites/suite_onboarding/shared/steps/signUpSteps.py:16]: This one is not needed I think.

**Review by endulab — APPROVED** [review]

File: test/ui-test/testSuites/suite_settings/tst_contactsFlow/test.feature [hunk 2]
```diff
@@ -11,6 +11,41 @@ Feature: Status Desktop Contacts Flows
...
     Scenario: The user can add a contact with a chat key
     	When the user opens the messaging settings
+    	And the user opens the contacts settings
     	And the user sends a contact request to the chat key "zQ3shQihZMmciZWUrjvsY6kUoaqSKp9DFSjMPRkkKGty3XCKZ" with the reason "I am a fellow tester"
         Then the contact request for chat key "zQ3shQihZMmciZWUrjvsY6kUoaqSKp9DFSjMPRkkKGty3XCKZ" is present in the pending requests tab
-        # TODO for future improvements: log into the other account and check that we received the request (will require some cleanup)
\ No newline at end of file
+        # TODO for future improvements: log into the other account and check that we received the request (will require some cleanup)
+
+    #relyon-mailserver
```
Comment (stefandunca) [inline test/ui-test/testSuites/suite_settings/tst_contactsFlow/test.feature:19]: Why not a tag?
```suggestion
    @relyon-mailserver
```

**Review by stefandunca — APPROVED** [review]
Excellent!

File: test/ui-test/testSuites/suite_settings/tst_contactsFlow/test.feature (hunk 2 above)
Comment (jrainville) [reply test/ui-test/testSuites/suite_settings/tst_contactsFlow/test.feature:19]: That's an oopsie

**Pull request merged**